src/content.js

```diff
@@ -258,6 +258,11 @@ chrome.storage.sync.get('layoutPreference', function (result) {
 		styleSheet.type = 'text/css';
 		styleSheet.innerText = LAYOUT_CHANGE;
 
+		if (result['layoutPreference'] === 'default-1080') {
+			styleSheet.innerText +=
+				'\n#main-wrap {--main-max-width: 1500px !important;}';
```

Comment:
```suggestion
				'\nbody #main-wrap {--main-max-width: 1500px !important;}';
```
Could you make the selector here more specific so that the other rule with 2500px is also overridden in Firefox?

Reply:
Accepted suggestion - Works on Chrome and Firefox now.